politeiawww/user/cockroachdb/cockroachdb.go

```diff
@@ -280,6 +280,61 @@ func (c *cockroachdb) UserGetByPubKey(pubKey string) (*user.User, error) {
 	return usr, nil
 }
 
+func (c *cockroachdb) UsersGetByPubKey(pubKeys []string) (
```

Comment:
Exported functions require a comment in the form of:

`// UsersGetByPubKey ...`

Reply:
Also, function declarations should be a single line.